en/toc.rst

```diff
@@ -21,6 +21,8 @@ The contents of the CUBRID Database Management System (CUBRID DBMS) product manu
 
 *   :doc:`admin/index`: This chapter provides instructions on how to create, drop, back up, restore and migrate a database, configuring globalization, and executing CUBRID HA. Also it includes instructions on how to use the  **cubrid** utility, which starts and stops the server, broker, and CUBRID Manager server, etc. Also, this chapter provides instructions on setting system parameters that may influence the performance. It provides information on how to use the configuration file for the server and broker, and describes the meaning of each parameter.
```

Comment:
1. Also it includes... --> Also, (comma added)

2. Also, this chapter provides instructions on setting system parameters that may influence the performance. --> 
In addition, this chapter provides..,  ( you have use 'also' in the last sentence, probably would be better to change 'also' to another word such as 'in addition,')

Reply:
okay sorry didnt know that, then I guess there are no problem with other part.